### PR TITLE
Small fixes for Ubuntu installation

### PR DESCRIPTION
**Pull request by Quarub**

### CODE REVIEW

File: callchain_checker/src/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 add_library(ccc-lib CallChainChecker.cpp)
+target_include_directories(ccc-lib PUBLIC ${LLVM_INCLUDE_DIRS} ${CLANG_INCLUDE_DIRS})
 target_link_libraries(ccc-lib PUBLIC clang-cpp Boost::headers)
 
 add_executable(ccc CallChainCheckerTool.cpp)
+target_include_directories(ccc PRIVATE ${LLVM_INCLUDE_DIRS} ${CLANG_INCLUDE_DIRS})
```
Comment (thetheodor) [inline callchain_checker/src/CMakeLists.txt:6]: Btw, this leaves in here now: https://github.com/DeadCodeProductions/callchain-checker and provides a python wrapper

File: init.py [hunk 2]
```diff
@@ -150,7 +150,10 @@ def main() -> None:
         # $ csmith --version csmith 2.3.0
         # Git version: 30dccd7
         version = res.split("\n")[0].split()[1]
-        csmith["include_path"] = "/usr/include/csmith-" + version
+        if Path("/usr/include/csmith").exists():
+            csmith["include_path"] = "/usr/include/csmith"
+        else:
+            csmith["include_path"] = "/usr/include/csmith-" + version
```
Comment (thetheodor) [inline init.py:156]: Maybe we can move this code to diopter's csmith generator?

**Review by thetheodor — APPROVED** [review]

File: callchain_checker/src/CMakeLists.txt (hunk 1 above)
Comment (Quarub) [reply callchain_checker/src/CMakeLists.txt:6]: Didn't use it due to the dependency on diopter and I thought we are doing a rewrite anyway :D

File: init.py (hunk 2 above)
Comment (Quarub) [reply init.py:156]: :+1:
A bit more validation on `diopter`s side might be nice. It already bit one user.

**Pull request merged**